Add a space in shell script

https://github.com/Juniper/apstra-go-sdk/blob/a48e304bd4b249c962d5cb45c294597e65b74cec/.ci/scripts/license_header_check.sh#L55

Should be `$( (git` rather than `$((git`

See [SC1102](https://www.shellcheck.net/wiki/SC1102)

--- a/.ci/scripts/license_header_check.sh
+++ b/.ci/scripts/license_header_check.sh
@@ -52,7 +52,7 @@ do
     else
       first_year="$line"
     fi
-  done <<< "$((git log --follow --pretty=format:"%ad" --date=format:'%Y' "$file"; echo) | sed -n '1p;$p')"
+  done <<< "$( (git log --follow --pretty=format:"%ad" --date=format:'%Y' "$file"; echo) | sed -n '1p;$p' )"
 
   # assume current year for both values if git log didn't find anything (new file)
   [ -z "$first_year" ] && first_year=$(date '+%Y')

